src/index.html

```diff
@@ -14,7 +14,7 @@
       content="Try Artichoke! The Artichoke Playground is a WebAssembly frontend for Artichoke Ruby."
     />
     <meta property="og:type" content="website" />
-    <meta property="og:image" content="https://artichoke.run/logo.svg" />
+    <meta property="og:image" content="https://github.com/artichoke/logo/raw/cbd027fd403e32c443962c66c5ca7a840a8528b8/logo.png" />
```

Comment:
I'm glad the png logo works. I'd prefer that the assets in the playground were self-contained instead of shelling out to GitHub.

Can we copy the png into this repo and build it into the deployed artifact with webpack?

We should be able to add another special case like `playground.png` here: https://github.com/artichoke/playground/blob/51d5500bd7b9f2fd75859d7ed656b13c92fab322/webpack.config.js#L71-L89